instructure/canvas-ios
# Add ability for teachers to allow users to comment on announcements

refs: MBL-10100
affects: Teacher
release note: Added a field for announcement creation and update to allow teachers to toggle allowing user to comment on announcements

test plan:
 - As a teacher create an announcement with the allow users to comment
   toggle off
 - Open the announcement in the student app and they shouldn't be able to reply
 - Edit the announcement and toggle to allow users to comment
 - Open the announcement in the student app and they should be able to reply
